spec/models/apartments_spec.rb

```diff
@@ -1,7 +1,51 @@
 require 'rails_helper'
 
 RSpec.describe Apartment, type: :model do
-  it("Apartment model exists") do
-  expect{Apartment.create}.to_not raise_error 
+
+  it 'is not valid without a street' do
+    Sza = Apartment.create street: '', city: 'New Orleans', state: "The Boot", manager: 'My own Boss', email: 'szaseason@yahoo.com', price: 'unlimited', pets: '2 small dogs.', bedrooms: '7', bathrooms: '7', image: "N/A", user_id: 'SzaSeason' 
+    # User = Users.create email: 'szaseason@yahoo.com', encrypted_password: 'HotGirlSummer305'
+     expect(Sza.errors[:street]).to_not be_empty
+    end
+    it 'is not valid without a city' do
+      Sza = Apartment.create street: 'Canal ST', city: '', state: "The Boot", manager: 'My own Boss', email: 'szaseason@yahoo.com', price: 'unlimited', pets: '2 small dogs.', bedrooms: '7', bathrooms: '7', image: "N/A", user_id: 'SzaSeason'
```

Comment:
Make sure that the datatypes of the columns match what you are creating with.  A good way to compare whether you should be using a string vs an integer is by looking at the schema.